packages/frontend/apps/electron/src/main/recording/state-machine.ts

```
@@ -22,56 +20,33 @@ export type RecordingEvent =
     }
+  | { type: 'FINALIZE_RECORDING_FAILED'; id: number; errorMessage?: string }
+  | { type: 'ABORT_RECORDING'; id: number }
   | { type: 'REMOVE_RECORDING'; id: number };
 
-/**
- * Recording State Machine
- * Handles state transitions for the recording process
- */
 export class RecordingStateMachine {
   private recordingId = 0;
   private readonly recordingStatus$ =
```

ESLint

[error] Finnish notation should not be used here.

(rxjs/finnish)